frontend/src/app/[locale]/process/ProcessIntro.tsx

```diff
@@ -1,7 +1,5 @@
 import { useMessages, useTranslations } from "next-intl";
-import { Grid } from "@trussworks/react-uswds";
-
-import ContentLayout from "src/components/ContentLayout";
```

Comment:
@andycochran @doug-s-nava we discussed removing `<ContentLayout/>` so didn't reproduce here.